pom.xml

```diff
@@ -247,6 +247,8 @@
               <goal>shade</goal>
             </goals>
             <configuration>
+              <shadedArtifactAttached>true</shadedArtifactAttached>
+              <shadedClassifierName>deploy</shadedClassifierName>
```

Comment:
I'm not a fan of that name. How about `jar-with-dependencies` like the `maven-assembly-plugin`'s [predefined descriptor](https://maven.apache.org/plugins/maven-assembly-plugin/descriptor-refs.html#jar-with-dependencies) would do?
(which is also why I suggested using the `maven-assembly-plugin` here)

Or maybe `all-deps` or `alldeps`? or `standalone`?